code/src/components/Breathe.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export const Breathe = ({ name, activity, colour, place, temp, who }) => {
+  return (
+    <div className="content-container">
+      <svg width="180" height="180">
+        <rect
+          x="20"
+          y="20"
+          width="170"
+          height="170"
+          fill="none"
+          strokeDasharray="20 20" />
+        <circle cx="10" cy="10" r="8" fill={colour} className="blurryCircle">
```

Comment:
This was really the icing on the cake! Managing to implement a moving image that is also relevant to the excercise, amazing!